api/alarms/background.js

```diff
@@ -0,0 +1,11 @@
+// Open the extension's demo page on install/update
+chrome.runtime.onInstalled.addListener((_reason) => {
+  chrome.tabs.create({
+    url: chrome.runtime.getURL('index.html')
```

Comment:
I'm a bit confused about how someone gets back to this page a 2nd time. Should there be an action to open the page? Can the link be included on the extensions' settings page?

Reply:
I skipped over that use case. It would be easiest to open a new tab on action click, so I'll run with that for the moment. 

This touches on a broader question, though: should we establish a set of conventions for extensions demos? I've opened #590 to kick around some ideas on this front.